src/interface/src/styleguide/search-result-card/search-result-card.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, Input, EventEmitter, Output } from '@angular/core';
+import { NgClass, NgFor, NgIf } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatExpansionModule } from '@angular/material/expansion';
+import { TreatmentTypeIconComponent } from '../treatment-type-icon/treatment-type-icon.component';
+import { SequenceIconComponent } from '../sequence-icon/sequence-icon.component';
+import { TreatmentProjectArea } from 'src/app/types';
+import {
+  PRESCRIPTIONS,
+  PrescriptionSingleAction,
+  PrescriptionSequenceAction,
+} from '../../app/treatments/prescriptions';
+/**
+ * Search Result Card Component
+ * A component to be used in the left panel to highlight matching results
+ */
+@Component({
+  selector: 'sg-search-result-card',
+  standalone: true,
+  imports: [
+    MatExpansionModule,
+    MatIconModule,
+    NgClass,
+    NgIf,
+    NgFor,
+    SequenceIconComponent,
+    TreatmentTypeIconComponent,
+  ],
+  templateUrl: './search-result-card.component.html',
+  styleUrl: './search-result-card.component.scss',
+})
+export class SearchResultCardComponent {
+  @Input() projectArea!: TreatmentProjectArea;
+  @Input() searchString: string = '';
+  @Input() wholeWordsOnly: boolean = false;
+  @Output() cardClick = new EventEmitter();
+
+  extractProjectTitle() {
+    return this.projectArea.project_area_name;
+  }
+
+  handleClick(event: Event) {
+    this.cardClick.emit(true);
+  }
+
+  extractProjectLines(): string[] {
```

Comment:
Maybe there's an optimization we can do here, so we can avoid this running on each angular cycle... something like 

```
onInit(){
this.projectLines = this.extractProjectLines()
```

and avoid calling extractProjectLines() on the template. Or at least the first piece where we do `lines`